setup.py

```diff
@@ -19,8 +19,8 @@
       py_modules=['Chandra.axTime3', 'Chandra.Time'],
       version='3.20',
       zip_safe=False,
-      test_suite="Chandra.test_Time",
-      packages=['Chandra'],
-      package_dir={'Chandra': 'Chandra'},
+      test_suite="Chandra.Time.test_Time",
+      packages=['Chandra', 'Chandra/Time'],
```

Comment:
Should be `['Chandra', 'Chandra.Time', 'Chandra.Time.tests']`.

I don't think the `package_dir` directive is needed in this case but not 100% sure.    The `py_modules` directive is not needed now.  Hopefully moving `axTime3` into `Chandra/Time/` won't break/confuse everything.